common/v2/features/DeFiZap/components/ZapIndicators.tsx

```diff
@@ -15,19 +14,20 @@ interface SProps {
 const SContainer = styled.div`
   display: flex;
   justify-content: center;
-  align-items: top;
+  align-items: center;
 `;
 
 const SImage = styled.img`
-  height: 24px;
-  width: 24px;
+  height: 16px;
+  width: 16px;
 `;
 
-const SText = styled(Typography)`
+const SText = styled.div`
```

Comment:
You can do `Typography` with attribute `as=div` if needed.

Reply:
This one was part of Guillaume's styling fixes, so I'm not going to touch it on the off chance it triggers another design review :3